src/cider/nrepl.clj

```diff
@@ -231,7 +231,6 @@ Depending on the type of the return value of the evaluation this middleware may
    "ns"      "The current namespace"
    "context" "A Compliment completion context, just like the ones already passed for the \"complete\" op,
 with the difference that the symbol at point should be entirely replaced by \"__prefix__\".
-
```

Comment:
That's problematic about it?

Reply:
https://docs.cider.mx/cider-nrepl/nrepl-api/ops.html#info

There are these broken bullet points

<img width="767" alt="image" src="https://github.com/clojure-emacs/cider-nrepl/assets/1162994/01a07445-b3fe-421e-a7d8-4ffe94b61331">